src/mlagility/analysis/analysis.py

```diff
@@ -119,30 +121,34 @@ def call_benchit(
 
     perf = None
     try:
-        perf = benchmark_model(
-            model_info.model,
-            inputs,
-            device=tracer_args.device,
-            backend=tracer_args.backend,
-            runtime=tracer_args.runtime,
-            build_name=build_name,
-            cache_dir=tracer_args.cache_dir,
-            build_only=Action.BENCHMARK not in tracer_args.actions,
-            lean_cache=tracer_args.lean_cache,
-            groq_num_chips=tracer_args.groq_num_chips,
-            groq_compiler_flags=tracer_args.groq_compiler_flags,
-            groq_assembler_flags=tracer_args.groq_assembler_flags,
-            groqview=tracer_args.groqview,
-            sequence=tracer_args.sequence,
-            onnx_opset=tracer_args.onnx_opset,
-        )
-
-        if Action.BENCHMARK in tracer_args.actions:
-            model_info.status_message = "Model successfully benchmarked!"
-            model_info.performance = perf
+        if model_info.model_type == build.ModelType.PYTORCH_COMPILED:
+            model_info.status_message = "Skipping model compiled using torch.compile(). Benchit requires models to be in eager mode."
```

Comment:
What if `runtime` is set to `torch-compiled`? Is the idea that `torch-compiled` only works with eager mode models that we compile ourselves?

Reply:
```suggestion
            model_info.status_message = "Skipping model compiled using torch.compile(). benchit requires models to be in eager mode (regardless of what runtime you have selected)."
```

little "b" for benchit

Reply:
Changed!